.github/actions/go-test/entrypoint.sh

```diff
@@ -19,27 +19,53 @@
 set -e
 
 download_go() {
+	. build/functions.sh
+	if verify_and_set_go_version; then
+		return
+	fi
 	go_version="$(cat "${GITHUB_WORKSPACE}/GO_VERSION")"
 	wget -O go.tar.gz "https://dl.google.com/go/go${go_version}.linux-amd64.tar.gz"
-	tar -C /usr/local -xzf go.tar.gz
+	echo "Extracting Go ${go_version}..."
+	<<-'SUDO_COMMANDS' sudo sh
+		set -o errexit
+		go_dir="$(
+			dirname "$(
```

Comment:
wouldn't simple piping work?

Reply:
It would, but be592797c5 removes this part.